docs/content/how-tos/rules/FL0082.md

```diff
@@ -0,0 +1,30 @@
+---
+title: FL0082
+category: how-to
+hide_menu: true
+---
+
+# UsedUnderscorePrefixedElements (FL0082)
+
+*Introduced in `0.21.8`*
```

Comment:
@webwarrior-ws please fix this to say 0.23.0

Reply:
updated version number to 0.23.0